vllm/config.py

```diff
@@ -971,7 +971,8 @@ def __init__(self,
                  is_multimodal_model: bool = False,
                  preemption_mode: Optional[str] = None,
                  num_scheduler_steps: int = 1,
-                 send_delta_data: bool = False) -> None:
+                 send_delta_data: bool = False,
+                 policy: str = "priority") -> None:
```

Comment:
Default should be fcfs right?
```suggestion
                 policy: str = "fcfs") -> None:
```